### PR TITLE
Qiskit for Quantum Computing

### LINKED ISSUE #6803: [Machine Learning] Getting Started with Qiskit for Quantum Computing
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Getting Started with Qiskit for Quantum Computing

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Qiskit is an acronym for `Quantum Information Software Kit for Quantum Computation`. It is an open-source software development kit for working with quantum computers at the level of circuits, pulses, and algorithms. It provides tools for creating and manipulating quantum programs and running them on prototype quantum devices on IBM Quantum Experience or on simulators on a local computer.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader will be able to understand:

- What is Qiskit
- How to install Qiskit
- What Qiskit can do.
- How to implement a simple Qiskit code example

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

Quantum computers haven't been with us for a long time. Most people have heard of it but never really tried using it. With open-source toolkit such as Qiskit, I'll be able to introduce our readers to the toolkit and let them experiment using Jupyter Notebooks hosted in IBM Quantum Lab. 

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

- [Qiskit](https://qiskit.org/)

### PR DESCRIPTION
**Pull request by monicada**
This closes #6803

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

File: content/articles/qiskit-for-quantum-computing/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+### How to install Qiskit
+To install qiskit, you have two options. You can install it locally on your local computer or on the Jupyter Notebook hosted by IBM. To install locally on a Linux machine, write the following code:
+
+```bash
+pip install -U pip && pip install qiskit
+```
+> This installation is for Linux machines. For Windows and macOS users, head on to the main website and select the appropriate machine you're using. A different installation code will be generated. Python version 3.6 and above is preferred.
+
+To get started online at IBM, we need to sign up [here](https://lab.quantum-computing.ibm.com) to get the API token which gives us permission to use their quantum devices. You can think of this are running Jupyter Notebook online with Qiskit installed.
```
Comment (Bayler) [inline content/articles/qiskit-for-quantum-computing/index.md:68]:
```suggestion
To get started online at IBM, we need to sign up [here](https://lab.quantum-computing.ibm.com) to get the API token that permits us to use their quantum devices. You can think of this are running Jupyter Notebook online with Qiskit installed.
```

File: content/articles/qiskit-for-quantum-computing/index.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+> Remember your API token is your access code to IBM's quantum devices.
+
+You can get it by going to your `Profile` option on the top-right corner of the webpage and clicking the `Account details` option. In the API token section, you'll your API token hidden from view in asterics. Copy it and paste it inside the following code:
```
Comment (Bayler) [inline content/articles/qiskit-for-quantum-computing/index.md:72]:
```suggestion
You can get it by going to your `Profile` option on the top-right corner of the webpage and clicking the `Account details` option. In the API token section, you'll get your API token hidden from view in `***`. Copy it and paste it inside the following code:
```

File: content/articles/qiskit-for-quantum-computing/index.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+  - url: /engineering-education/qiskit-for-quantum-computing/hero.png 
+    alt: Qiskit for Quantum Computing Hero image
+---
+For most of us, it's hard to wrap our heads around the world of quantum computing. Most people have heard of it, but a lot of information about it glosses over important details. 
+<!--more-->
+This tutorial aims to clear this up and introduce you to Quantum computing. With the help of an open-source toolkit such as Qiskit, we'll introduce our readers to the toolkit and let them experiment using Jupyter Notebooks hosted in IBM Quantum Lab. 
+
+### Quntum computers
+Quantum computers solve problems in a different way from the classic computers that we are familiar with. Quantum computers have several advantages over classical computers for certain problems. This is attributed to their ability to exist in a number of different states simultaneously. On the contrary, classical computers can only be in one state at a time.
```
Comment (Bayler) [inline content/articles/qiskit-for-quantum-computing/index.md:22]:
```suggestion
Quantum computers solve problems differently from the classic computers that we are familiar with. Quantum computers have several advantages over classical computers for certain problems. This is attributed to their ability to exist in several different states simultaneously. On the contrary, classical computers can only be in one state at a time.
```

**Comment by ahmadmardeni1** [comment]
Great work team @monicada @LinusMuema

**Pull request merged**